deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -97,7 +97,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: hostpath
-          image: k8s.gcr.io/sig-storage/hostpathplugin:v1.7.2
+          image: k8s.gcr.io/sig-storage/hostpathplugin:v1.8.0
```

Comment:
Let's wait for a new hostpathplugin image.

Reply:
sure.